packages/flutter/test/painting/text_painter_rtl_test.dart

```diff
@@ -284,25 +286,31 @@ void main() {
       painter.getOffsetForCaret(const TextPosition(offset: 1, affinity: TextAffinity.upstream), Rect.zero),
       const Offset(10.0, 0.0),
     );
+
+    Offset offset = (Platform.isWindows) ? new Offset(7.0, 12.0) : new Offset(10.0, 10.0);
```

Comment:
final for `offset` and const for `Offset`?

Reply:
what is going on here? This seems extremely wrong. The Ahem font is exactly 1em square for every glyph. If this is not coming out as 10,10, then there's a real bug.

Reply:
@goderbauer, using const for `Offset` doesn't seem to work since `Platform.isWindows` isn't const.

@Hixie, I'm not 100% sure as I'm not familiar with this code. However, I was running into issues with Windows rendering fonts differently than other platforms which broke tests that were looking for specific pixel values, so I assumed that was the case here.

Reply:
This is what I meant, this doesn't work?

```dart
final Offset offset = (Platform.isWindows) ? const Offset(7.0, 12.0) : const Offset(10.0, 10.0);
```

Reply:
The analyzer won't let you do anything but what @goderbauer suggests. :-)

Reply:
Hm, I'll have to check again... I might have been trying something like this:

```dart
const Offset offset = (Platform.isWindows) ? const Offset(...) : const Offset(...);
```